dirsrvtests/tests/tickets/ticket2058_test.py

```diff
@@ -0,0 +1,131 @@
+import logging
+import pytest
+import os
+import subprocess
+import time
+import ldap
+from lib389._constants import *
+from lib389.topologies import topology_i2 as topo
+
+DEBUGGING = os.getenv("DEBUGGING", default=False)
+if DEBUGGING:
+    logging.getLogger(__name__).setLevel(logging.DEBUG)
+else:
+    logging.getLogger(__name__).setLevel(logging.INFO)
+log = logging.getLogger(__name__)
+
+def run(cmd):
+  rc = subprocess.run(cmd.split(), capture_output=True)
+  if (rc.returncode != 0):
+      log.error(f"Command {cmd} failed: {rc}")
+      assert False
+  return rc
+
+def check_single_keep_alive(topo,instance,dn,expected):
+    # check that keep alive entry does not exists on master1
+    try:
+        ent = topo.ins[instance].getEntry(dn, ldap.SCOPE_BASE, "(objectclass=ldapsubentry)", ['nsUniqueId', 'modifierTimestamp'])
+    except ldap.NO_SUCH_OBJECT:
+        if (expected):
+            log.error('Keepalive entry %s does not exists on instance %s' % (dn, instance))
+            assert False
+        else:
+             log.debug('As expected, keepalive entry %s does not exists on instance %s' % (dn, instance))           
+    except ldap.LDAPError as e:
+        log.fatal('Failed to retrieve keepalive entry (%s) on instance %s: error %s' % (dn, instance, e.message['desc']))
+        assert False
+    else:
+        if (expected):
+            log.debug('Found keepalive entry %s on instance %s' % (dn, instance))
+            log.debug('Keep alive entry  is: ' + str(ent));
+        else:          
+            log.error('Unexpectedly found keepalive entry %s on instance %s' % (dn, instance))
+            log.info('Keep alive entry  is: ' + str(ent));
+            assert False
+
+def check_keep_alive(topo, expected):
+    check_single_keep_alive(topo, "standalone1", f"cn=repl keep alive 1, {SUFFIX}", expected)
+    check_single_keep_alive(topo, "standalone1", f"cn=repl keep alive 2, {SUFFIX}", expected)
+    check_single_keep_alive(topo, "standalone2", f"cn=repl keep alive 1, {SUFFIX}", expected)
+    check_single_keep_alive(topo, "standalone2", f"cn=repl keep alive 2, {SUFFIX}", expected)
+
+
+def test_ticket2058(topo):
+    """Checks that replication keep alive entry are created after on line initialisations
+       (without the fix, only the entry from master1 is created)
+
```

Comment:
Also, add additional field here - `:id: UUID`.
You can get UUID like this: `python3 -c "import uuid; print(uuid.uuid4())"`

Reply:
Thanks !
I now understand that these :id:  are random uuid.  Ok I will gerate them using  uuid -m -v 4